eZ/Publish/Core/MVC/Symfony/Controller/Content/DownloadController.php

```diff
@@ -75,7 +75,7 @@ public function downloadBinaryFileAction($contentId, $fieldIdentifier, $filename
         }
 
         $response = new BinaryStreamResponse($this->ioService->loadBinaryFile($field->value->id), $this->ioService);
-        $response->setContentDisposition(ResponseHeaderBag::DISPOSITION_ATTACHMENT, $filename);
+        $response->setContentDisposition(ResponseHeaderBag::DISPOSITION_ATTACHMENT, $filename, md5($filename));
```

Comment:
SonarLint reports here a security issue. It's false negative, but to avoid any discussion is it technically possible here to use some better algorithm?

Reply:
We can use anything here really, but technically this doesn't have to be secure. It's rather a matter of encoding.